<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>정렬</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dataTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>기관명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>간접비율</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>%</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="actionfile_open"/>
    <addaction name="actionfile_save"/>
   </widget>
   <addaction name="menuopen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionfile_open">
   <property name="text">
    <string>file open</string>
   </property>
  </action>
  <action name="actionfile_save">
   <property name="text">
    <string>file save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
